src/main/java/com/booksaw/betterTeams/commands/presets/TeamSubCommand.java

```diff
@@ -21,6 +22,11 @@ public abstract class TeamSubCommand extends SubCommand {
 	protected boolean checkRank = true;
 	PlayerRank requiredRank = getDefaultRank();
 
+	protected @Nullable Team getMyTeam(CommandSender sender) {
+		Player player = (Player) sender;
```

Comment:
Firstly what is the point in this method, it just calls an existing method with no additional processing or results caching.

I would suggest keeping this method, caching the result in a local private variable to save repeated "Team.getTeam" calls and also using this method in the `onCommand` block below.

Additionally, I would suggest taking the type Player as a parameter rather than CommandSender

Reply:
Basically it's a preparation for another PR I'm working on 😉. But caching it is a good idea. I'll do that!

Reply:
I was actually thinking about replacing it in the lines beneath it, but I decided against it because it's only one line, and the `player` variable is also necessary in that method.

Secondly, I was thinking about passing a Player instead, but then you would constantly need to cast in the calling of the function. Now it's hidden away inside the method.

Reply:
Ok, added the caching.